torchtext/experimental/utils/samplers/seqlen_sampler.py

```diff
@@ -0,0 +1,73 @@
+from typing import List
```

Comment:
I think we can have a `simpler.py` file in `experimental/utils` folder and have all the samplers there.